### ISSUE TITLE
Can this project be used for non diversity use?

### ISSUE THREAD

**Issue by llambkin** [gc9n/rx5808-Diversity-RealAcc-Compatible#13]
Hey gc9n,

I am using your project on a custom setup I will be using in some Fatshark Base SD goggles. I want the search function and OSD RSSI infos on my screen, but when I go to compile it with USE_DIVERSITY undefined, it kicks up a fuss and spits this up as an error in Arduino:

rx5808-pro-diversityGc9n:1463: error: 'rssiB' was not declared in this scope

       OSDParams[1] = rssiB; //rssiB; //this is  B

                      ^

rx5808-pro-diversityGc9n:1464: error: 'receiver' was not declared in this scope

       OSDParams[2] = receiver; //active_receiver; //CUrrent reciever

                      ^

exit status 1
'rssiB' was not declared in this scope

Also I have compiled it with diversity enabled, and I am getting some weird flashing on the screen when it displays the channel searching. I dont know if this is related to having diversity enabled or not.

My setup is as follows:
Arduino Nano with gc9n firmware
MinimOSD with fonts installed and Polyphemus firmware
RX5808 bare module SPI mod
At the moment, it is just hooked up on the desk until I can get it functioning correctly, then it will go in the goggles.

**Comment by llambkin** [comment]
Also forgot to mention, I am not using a screen either. I was going to rely solely on the OSD to control the channels.

**Comment by gc9n** [comment]
You better define USE_DIVERSITY
And then set the reciever a.  What I mean is that diversity have to be on a not on auto

**Comment by llambkin** [comment]
Any idea what would cause this? It only happens after a while, not straight away.
![img_20170208_203727](https://cloud.githubusercontent.com/assets/3177909/22737628/d338b1a0-ee3e-11e6-8c18-151f507be872.jpg)

**Comment by gc9n** [comment]
there is a bug in auto frequensy.
ingore it

**Comment by llambkin** [comment]
Awesome. Thanks for replying. I'm guessing that it is being worked on?
Trying not to sound ungrateful for your awesome work. I looove what you
have achieved with this project. I can't wait to see how much further you
can take this.

**Comment by llambkin** [comment]
![14865592817701346618642](https://cloud.githubusercontent.com/assets/3177909/22738515/b486a286-ee42-11e6-95a2-849300915a0f.jpg)

**Comment by gc9n** [comment]
thats fuckin... great Man!! put it in a box , and your ready

**Comment by llambkin** [comment]
![14865595114941474218700](https://cloud.githubusercontent.com/assets/3177909/22738648/3c192408-ee43-11e6-9e21-02db9a81ab3f.jpg)

**Comment by gc9n** [comment]
i have those goggles also. its the RCV922

**Comment by gc9n** [comment]
if i where you, i wouldnt use the Arduino NANO , but the arduino pro mini .. because is small.
and draws less current

**Comment by gc9n** [comment]
put it in a nice box upon (externally) like a module

**Comment by llambkin** [comment]
Yeah, I did that with my Doms, I had the Nano lying around and I thought Id make use of it.
I just had to do this when sheaivey released his code:
https://www.rcgroups.com/forums/showthread.php?2629411-Modified-Fatshark-Dominator-V2-s-with-OLED-screen
I like the stock look.

**Comment by gc9n** [comment]
wow nice job man..

**Comment by llambkin** [comment]
Thanks. Let me know if you are working on anything. I am more than willing to pull some stuff apart to help you with alpha testing. I quite often frequent the Betaflight slack group, testing all sorts of alpha code.

**Comment by llambkin** [comment]
So.... It fits! It works ok, no Vbat, need some sleep, will tackle it tomorrow.

![img_20170208_235849](https://cloud.githubusercontent.com/assets/3177909/22746484/c4701418-ee5e-11e6-9494-636d32789ba2.jpg)
